### DISCUSSION TITLE
why is there version discrepancy

### DISCUSSION THREAD

**Discussion by mcwienczek-rtl** [Kong/kong#8897] [Help] [answered]
Seems there is some kind of confusion around versions of kong.

Github says that latest kong is 2.4.2 released 11 days ago. But CHANGELOG.md from master says that latest version is 2.8.0.

Fun fact: version 2.4.2 is not present in CHANGELOG.md. 

Where should we look for newest releases?

**Comment by jschmid1** [comment, marked as answer]
You can find _all_ releases [here ](https://github.com/Kong/kong/releases). The 2.4.2 just happened to be the latest release and is hence at the top (due to chronological sorting).

> Fun fact: version 2.4.2 is not present in CHANGELOG.md.

This is due to our branching policy. [`patch`  releases](https://semantic-release.gitbook.io/semantic-release/) are not forward-ported back into master. This is why the 2.4.2 changelog entry is missing.
